build/ci/release.yml

```diff
@@ -74,6 +74,7 @@ functions:
           - goreleaser_config
           - github_token
           - unstable
+          - segment_key
```

Comment:
`segment_key` must be defined as evergreen variant in the mongocli evergreen project